mesh/mesh.go

```diff
@@ -48,17 +52,17 @@ func (m *mesh) IsContexuallyValid(b BlockID) bool {
 	return true
 }
 
-func (m *mesh) LatestIrreversible() uint32 {
-	return atomic.LoadUint32(&m.latestIrreversible)
+func (m *mesh) LocalLayer() uint32 {
```

Comment:
what does local mean?

Reply:
tortoise is gonna add some counters it will be sorted as part of the next tortoise pr